app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js

```diff
@@ -76,14 +76,23 @@ async function addEthereumChainHandler(
     );
   }
 
+  const isLocalhost = (strUrl) => {
+    const url = new URL(strUrl);
+    return url.hostname === 'localhost' || url.hostname === '127.0.0.1';
```

Comment:
This PR looks good.  The only thing I'd mention is that `new URL` can throw an error if the input is a malformed URL, which is why we usually `try/catch` the `new URL` line.  Of course a malformed URL should never make it here, so maybe I'll see what others think.

Reply:
Done, thanks for the review!